src/cloud/tags.yml

```
@@ -2,13 +2,33 @@ tags:
   - name: Authorizations
     description: |
       Create and manage _API tokens_.
+
       An _authorization_ contains a list of `read` and `write`
       permissions for organization resources and provides an API token for authentication.
       An authorization belongs to an organization and only contains permissions for that organization.
 
-      An authorization is only visible to the user that created it.
+      In InfluxDB Cloud, an authorization with `read-authorizations` permission
+      can be used to view other authorizations.
       Optionally, when creating an authorization, you can scope it to a specific user.
-      A _user session_ carries the permissions granted by all the user's authorizations.
+
+      #### Limitations
+
+      To follow best practices for secure API token generation and retrieval,
+      InfluxDB Cloud enforces access restrictions on API tokens.
+
+      - InfluxDB Cloud only allows access to the API token value immediately after the authorization is created.
+      - You can’t change access (read/write) permissions for an API token after it’s created.
```

Just a suggestion, you don't have to use this if it doesn't fit better
```suggestion
      - Access (read/write) permissions for an API token can't be changed after it's created.
```